.vscode/launch.json

```diff
@@ -33,8 +29,8 @@
             "externalConsole": false,
             "MIMode": "gdb",
             "setupCommands": [ ],
-            "preLaunchTask": "CPP - Build and Run daprized App",
-            "postDebugTask": "dapr-VehicleApp-stop",
+            "preLaunchTask": "dapr-sidecar-start",
```

Comment:
The clean launch of the application is not working. The build is required in front. 
We need to define if build will be performed automatically before launch or after devcontainer created

Reply:
I've update README regarding build.

Reply:
It was absents of build.